templates/vue/src/components/lightbox/quiz/Question.vue

```diff
@@ -24,19 +24,22 @@
         <div class="button-container">
           <answer-button
             v-if="hasId('textId')"
+            :completed="true"
```

Comment:
Mocking data here only. Will need to pass in the real value. So this PR is still WIP.